frontend/src/views/SQR/Form.vue

```diff
@@ -193,13 +195,15 @@ export default class Form extends Vue {
   submitSuccess = false;
   formHasError = false;
   errorMessage = "";
-  isCoordinator = false;
   showValidationSummary = false;
-  getUserClaims!: any;
   rules = [required];
   destroyFormErrorWatch!: any;
   branch: SubmissionsBranch | null = null;
 
+  get sqrForm() {
+    return new SQRForm({ isCompleted: this.isCompleted });
+  }
```

Comment:
I feel it might be a bit confusing to have this property. This component is an SQR Form itself. Can't we just pass its data to the ability without introducing this property and even without SQRForm class?

Reply:
Maybe it should be renamed to something better but this is the intended way to pass in data. https://stalniy.github.io/casl/abilities/2017/07/20/define-abilities.html#rule-conditions

Reply:
I'm not about passing the data, but about the fact that `SQR/Form` component contains `sqrForm` property which is intended only for passing to the rule. Looks weird.

More weird is that the `SQRForm` class is defined in the ability definition. Consider refactoring this. Maybe you can pass the whole form to the rule if it's easy?